src/main/java/frc/utils/digitalinput/channeled/ChanneledDigitalInput.java

```diff
@@ -0,0 +1,31 @@
+package frc.utils.digitalinput.channeled;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+import org.littletonrobotics.junction.Logger;
+
+public class ChanneledDigitalInput implements IDigitalInput {
+
+	private final DigitalInput digitalInput;
+	private final Debouncer debouncer;
+	private final boolean inverted;
+
+	public ChanneledDigitalInput(int channel, double debounceTime, boolean inverted) {
```

Comment:
```suggestion
	public ChanneledDigitalInput(int channel, double debounceTime, boolean `returnFalseAsDefault`) {
```

maybe a bit overkill but I think it'll clarify some stuff.